app/src/main/java/com/omelan/cofi/pages/settings/licenses/DependencyItem.kt

```diff
@@ -0,0 +1,183 @@
+package com.omelan.cofi.pages.settings.licenses
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.*
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.omelan.cofi.pages.settings.settingsItemModifier
+import com.omelan.cofi.utils.URL_ANNOTATION
+import com.omelan.cofi.utils.addLink
+import org.json.JSONArray
+import org.json.JSONObject
+
+data class License(val license: String, val license_url: String)
+
+data class Dependency(
+    val project: String,
+    val description: String,
+    val version: String,
+    val developers: List<String>,
+    val url: String?,
+    val year: String?,
+    val licenses: List<License>,
+)
+
+private fun JSONArray.toAuthorList(): List<String> {
+    val list = mutableListOf<String>()
+    (0 until this.length()).forEach {
+        list.add(this.getString(it))
+    }
+    return if (list.isEmpty()) listOf("Original author or authors") else list
```

Comment:
I've decided against translating this as 

1. Licenses, packages, and descriptions are only in English so it would look strange to have different language in one place
2. it would require passing context and that's not clean